src/ts/ConvivaAnalytics.ts

```diff
@@ -404,9 +404,8 @@ export class ConvivaAnalytics {
         break;
     }
 
-    // TODO: check if also should not report for post-roll (currently doesn't)
     if (!this.isValidSession()) {
-      // Don't report without a valid session (e.g. in case of a post-roll ad)
+      // Don't report without a valid session (e.g. in case of a pre-roll ad)
```

Comment:
please leave the comment as it was. In case of a post-roll ad the session already got shut down. So the old comment fits better.

Reply:
But also there is not yet a session in case of a pre-roll ad. I will adjust the comment accordingly.